tracer/parser.go

```diff
@@ -229,6 +230,16 @@ func (p *Parser) Finish() error {
 	return nil
 }
 
+func (p *Parser) getTags() map[string]string {
+	tags := maps.Clone(p.global.Config.Tags)
+
+	for k, v := range p.global.Stats.GetStatsTags() {
```

Comment:
this is not concurrency safe. if two requests in two concurrent goroutines try to access the same keys at the same time the loop goroutine is writing a value, Go will panic.